tests/manual-test-cases/Group19-ROBO/19-3-ROBO-VM-Placement.md

```diff
@@ -0,0 +1,41 @@
+Test 19-3 - ROBO - VM Placement
+=======
+
+# Purpose:
+To verify that the VM placement feature specified works as expected in a vSphere ROBO Advanced environment without DRS.
+The current placement strategy is to avoid bad host selection, instead of selecting the "best" possible host.
+
+# References:
+1. [vSphere Remote Office and Branch Office](http://www.vmware.com/products/vsphere/remote-office-branch-office.html)
+2. [VM Placement without DRS](https://github.com/vmware/vic/issues/7282)
+
+# Environment:
+This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation. This test should be executed in the following topologies and should have vSAN enabled.
+* 1 vCenter host with 3 clusters, where 1 cluster has 1 ESXi host and the other 2 clusters have 3 ESXi hosts each
+* 2 vCenter hosts connected with ELM, where each vCenter host has a cluster/host/datacenter topology that emulates a customer environment (exact topology TBD)
+
+See https://confluence.eng.vmware.com/display/CNA/VIC+ROBO for more details.
+
+# Test Steps:
+1. Deploy a ROBO Advanced vCenter testbed for both environments above
+2. Install the VIC appliance on vCenter
+3. Deploy containers that will consume resources predictably (e.g. the `progrium/stress` image)
+4. Measure ESX host metrics and gather resource consumption
+5. Create and run regular containers such as `busybox`
+6. Create and run enough containers to consume all available host resources
```

Comment:
now I'm thinking single host cluster... and this test is focused on testing the resource exhaustion case.
But in that case I'd expect another test that ensures we can consume the entire _cluster_ resource.

Reply:
My intent for this test is to target a particular cluster (multi-host or otherwise) - I'll make the steps clearer and specific.